crates/ra_syntax/src/ast.rs

```diff
@@ -218,7 +218,7 @@ fn test_doc_comment_multi_line_block_strips_suffix() {
 fn test_comments_preserve_trailing_whitespace() {
     let file = SourceFile::parse(
         r#"
-/// Representation of a Realm.   
+/// Representation of a Realm.
 /// In the specification these are called Realm Records.
 struct Realm {}"#,
```

Comment:
Let's flip to a non-raw string literal here: "/// Representation of a Realm.   \n/// In the specification these are called Realm Records."

Reply:
Cargo fmt had spoiled it :(
Should I really flip it or just restore and do not breathe on it?

Reply:
I'ts better to just solve this by flipping to a non-raw literal, yeah.